app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -114,14 +114,16 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     protected void createGoogleApiClient() {
-        mGoogleApiClient = new GoogleApiClient.Builder(this)
-                .addApi(Plus.API)
-                .addApi(Games.API)
-                .addApi(AppStateManager.API)
-                .addScope(Plus.SCOPE_PLUS_LOGIN)
-                .addScope(Plus.SCOPE_PLUS_PROFILE)
-                .addScope(Games.SCOPE_GAMES)
-                .build();
+        GoogleApiClient.Builder builder = new GoogleApiClient.Builder(this)
+                .addApiIfAvailable(AppStateManager.API);
+        if (PrefUtils.isSignedIn(this)) {
+            builder.addApiIfAvailable(Plus.API, Plus.SCOPE_PLUS_LOGIN, Plus.SCOPE_PLUS_PROFILE)
```

Comment:
With the scopes the play service asks for a user account, without scopes play services do not ask for login but we can still use the APIs for the PlusPersonDownloader and the news feed.